core/src/com/unciv/ui/screens/diplomacyscreen/OffersListScroll.kt

```diff
@@ -64,6 +65,7 @@ class OffersListScroll(
                 Strategic_Resource -> "Strategic resources"
                 Technology -> "Technologies"
                 WarDeclaration -> "Declarations of war"
+                OfferPeaceTreaty -> "Peace Treaty"
```

Comment:
I hadn't checked before you asked, but I see that it is already in template.properties. If you want me to change the name to Peace Treaties (Plural since this is the label for a dropdown) I can add that.

Reply:
No, this is fine :)